lib/compilation-prepare.sh

```diff
@@ -288,6 +288,9 @@ compilation_prepare()
 		sed -i '/source "drivers\/net\/wireless\/ti\/Kconfig"/a source "drivers\/net\/wireless\/rtl8189es\/Kconfig"' \
 		"$kerneldir/drivers/net/wireless/Kconfig"
 
+                # add support for 5.19.2
+                process_patch_file "${SRC}/patch/misc/wireless-rtl8189es-5.9.2.patch" "applying"
```

Comment:
Isn't **5.9.2** a typo here? There are several patches where "5.9.2" is used, so many I can't understand if it is intentional or a typo :smile_cat: 
I guess the whole compiles but maybe the patch names are misleading?

Reply:
copy paste focus absence typo ;) I will correct those patches in next commits.

btw. If you want to pick one to adjust? 88x2bu or 88x2cs :) It seems all drivers are broken

Reply:
Sorry @igorpecovnik , went away and then come back too late so could not help

Reply:
No worries.